src/shared/tpm2-util.c

```diff
@@ -1105,18 +1105,11 @@ static int tpm2_get_best_srk_template(Tpm2Context *c, TPMT_PUBLIC *ret_template)
                                "TPM does not support either SRK template L-1 (RSA) or L-2 (ECC).");
 }
 
-/* The SRK handle is defined in the Provisioning Guidance document (see above) in the table "Reserved Handles
- * for TPM Provisioning Fundamental Elements". The SRK is useful because it is "shared", meaning it has no
- * authValue nor authPolicy set, and thus may be used by anyone on the system to generate derived keys or
- * seal secrets. This is useful if the TPM has an auth (password) set for the 'owner hierarchy', which would
- * prevent users from generating primary transient keys, unless they knew the owner hierarchy auth. See
- * the Provisioning Guidance document for more details. */
-#define TPM2_SRK_HANDLE UINT32_C(0x81000001)
-
-/* Get the SRK. Returns 1 if SRK is found, 0 if there is no SRK, or < 0 on error. Also see
- * tpm2_get_or_create_srk() below. */
-static int tpm2_get_srk(
+/* Get the Tpm2Handle at the requested index. Returns 1 if found, 0 if the index is empty, or < 0 on
+ * error. Also see tpm2_get_srk() below; the SRK is a commonly used persistent Tpm2Handle. */
+int tpm2_get_handle(
```

Comment:
maybe be explicit? tpm2_get_handle_from_index()?

Reply:
Since the `tpm2_get_handle()` and `tpm2_get_handle_index()` functions are reciprocal functions, I changed the names to `tpm2_index_from_handle()` and `tpm2_index_to_handle()`, how does that sound?